install/amazon_rds/04_configure_the_collector_docker.mdx

```diff
@@ -44,21 +44,20 @@ Create your configuration file (e.g. named `pganalyze_collector.env`) with envir
 
 ## Handling Amazon Aurora clusters vs instances
 
-In the case of Amazon Aurora, the collector automatically resolves `cluster` endpoints to the underlying writer instance, and `cluster-ro` to the underlying reader instance:
+In the case of Amazon Aurora, the collector automatically resolves `cluster` endpoints to the underlying writer instance.
 
 ```
 DB_HOST=mydbcluster.cluster-123456789012.us-east-1.rds.amazonaws.com
 ```
 
-or
+This will only monitor the writer instance. If you also want to monitor a reader instance, you'll need to run another Docker container.
+Create a new configuration file (e.g. named `pganalyze_collector.replica.env`) and use the `cluster-ro` endpoint as the `DB_HOST`:
```

Comment:
Technically, you could use CONFIG_CONTENTS to monitor the whole cluster with one collector. We do mention that in [our docs](https://pganalyze.com/docs/collector/settings) (separately, should it be clearer?). I feel like we should mention it here for customers who would rather just run a single collector. How about something like
```suggestion
This will only monitor the writer instance. If you also want to monitor a reader instance, you'll need to use the [`CONFIG_CONTENTS`](https://pganalyze.com/docs/collector/settings) configuration mechanism and add another section with the `cluster-ro` endpoint as the `db_host`.

Alternately, you can run a separate Docker container to monitor each node. Create a new configuration file (e.g. named `pganalyze_collector.replica.env`) and use the `cluster-ro` endpoint as the `DB_HOST`.
```

Reply:
Mentioning about `CONFIG_CONTENTS` here is a good idea, but I think that still can be a secondary option and we can suggest running two Docker containers as a primary option.
Or, do we actually want to push people to use `CONFIG_CONTENTS`?

Reply:
Yeah, good point. I think running a single collector for multiple servers (especially when they're closely related like this) is more manageable, but the other way is a bit simpler. I'd lean to preferring `CONFIG_CONTENTS`, but no strong feelings.